includes/server-timing/hooks.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Hook callbacks used for Server Timing.
+ *
+ * @package performance-lab
+ *
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+
+/**
+ * Adds server timing to REST API response.
+ *
```

Comment:
```suggestion
 * Adds server timing to REST API response.
 *
 * @since n.e.x.t
 *
```

Add `@since` for new function.